src/pages/[lang]/imprint.astro

```diff
@@ -22,7 +22,7 @@ export const getStaticPaths = getStaticLangPaths;
       17034 Neubrandenburg
     </p>
     <h2>Kontakt</h2>
-    <p>E-Mail: blz-it@bs-nb.de</p>
+    <p>E-Mail: <a href="mailto:blz@kapplers.de">blz@kapplers.de</a></p>
```

Comment:
Why don't you use the Link component here?

Reply:
To avoid styling every single text-tag here, I wrapped the entire page in `prose` from `@tailwindcss/typography`. This, however, also adds default styling to links. If I were to add the `Link` component to this default styling, it would look like the following:

![Screenshot 2023-01-30 at 18 35 06 (2)](https://user-images.githubusercontent.com/58258541/215551711-c1dda79b-d3be-4bc6-9eb4-f21d6b3b2244.png)

This is probably not what we want here. The default styling looks like the following:

![Screenshot 2023-01-30 at 18 36 14 (2)](https://user-images.githubusercontent.com/58258541/215551868-4a974122-6576-4be8-9bfb-33d8b489b1d8.png)

This also fits the page quite well, in my opinion.

Reply:
Of course, I just wanted to ask